openapi.yaml

```diff
@@ -369,6 +359,23 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/JsonError'
+              examples:
+                runerror: 
+                  value: 
+                    error: "Error Retrieving Runs"
+                  summary: An error occured when trying to retireve the runs
+                pageerror:
+                  value: 
+                    error: "Error retrieving page"
+                  summary: An error occured when trying to return a results page
+                instanterror:
+                  value:
+                    error: "Error parsing Instant"
```

Comment:
What does 'Error parsing Instant' mean ? Perhaps in the 'to' field, or 'from' field, and have separate errors ?

Also, parsing of input errors should not be a 500, but a 400 bad input error .

Reply:
Can we have a different error for each field, so users know exactly which one  had the problem in ?